app/views/users/sessions/sign_in_with_email.html.erb

```diff
@@ -1,8 +1,7 @@
 <% content_for(:head) do %>
   <title><%= "Sign In | #{current_school.name}" %></title>
 <% end %>
-
-<div class="bg-gray-50 sm:py-10">
+<div class="h-full bg-gray-50 py-10 md:py-24">
```

Comment:
I think we should some add padding horizontally (px) for the container. It looks like hitting the edges of the screen on mobile and small devices in all the 5 pages. 🤔

Reply:
@yash-learner After going through the changes, the container's missing horizontal padding (spacing) is kept intentional as it would take more space in the mobile screen, resulting in the size shrink of actual interactive elements.